### DISCUSSION TITLE
Breezy Weather Start on boot option?

### DISCUSSION THREAD

**Discussion by whitewallman** [breezy-weather/breezy-weather#800] [Q&A] [answered]
Hello and thank you for Breezy Weather!
Is there an option to toggle Breezy to start on boot?

**Comment by papjul** [comment]
What is "Start on boot"? Can you link the documentation of this feature?

**Reply by whitewallman** [reply]
I don't have documentation. I'm referring to an option to allow Breezy Weather to start automatically when the device starts or reboots. The option may already be there, and I may be missing it.

**Reply by papjul** [reply]
What do you mean by "start automatically"?
- If you are talking about periodic background updates, they persist after reboot
- If you are talking about bringing back widgets/notifications/tiles, they should be back after reboot
- If you are talking about opening the app on boot, I'm not aware of any such Android feature. Do you have a link to an open source app that makes use of it?

**Reply by whitewallman** [reply]
I'm sorry, I was referring to the notification. I'll have to watch closer, but I believe the notification doesn't appear until the chosen update duration instead of shortly after a reboot.
![Screenshot_20240314-160543](https://github.com/breezy-weather/breezy-weather/assets/109869840/e10e526c-6d2c-408f-8ac6-bb2482e7e6af)

**Reply by papjul** [reply]
I can reproduce. Issue #801 opened

**Reply by whitewallman** [reply]
Thanks! Love what you've done with Breezy! 
Thank you again!

**Reply by papjul** [reply, marked as answer]
Fixed, will be in v5.1.4
